### PR TITLE
feat(Artfight): add presence

### PR DESCRIPTION
**Pull request by RisingSunLight42**
## Description 

Add a new presence for the Artfight website.

### Changelog

- Add "attack" handling with slideshows, showing as most informations as possible
- Add "character", "team", "info", etc... handling
- Use an enum Assets to display logo and some `smallImageKey`
- Use showCover and showButtons settings
- The website can show NSFW art images, I would like to know if I have to flag the presence as NSFW in the metadata file. I've already blocked the showing of NSFW images on the presence when you're viewing an NSFW attack.

_Note : Due to the website being really active only for one month in the year, I haven't managed to implement all the functionnalities I've wanted to add because the site changes during the event time, and then rollback to its actual state.
However, everything that is actually implemented works for both of these versions !_

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

artfight_proof_2

artfight_proof_1

artfight_proof_5

artfight_proof_4

artfight_proof_3

</details>

### CODE REVIEW

**Review by EncryptedDev — APPROVED** [review]

**Comment by EncryptedDev** [comment]
Nice use of the slideshow feature

**Comment by RisingSunLight42** [comment]
> Nice use of the slideshow feature

Thanks! I'm trying to use it for good purposes, happy to see that I've managed to use it well 👌

File: websites/A/Artfight/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "RisingSunLight",
+		"id": "240521747852558347"
+	},
+	"service": "Artfight",
+	"description": {
+		"en": "An art trading game"
```
Comment (Bas950) [inline websites/A/Artfight/dist/metadata.json:9]:
```suggestion
		"en": "The annual art trading game, starting on July 1st!"
```

**Review by Bas950 — COMMENTED** [review]
this might be better

File: websites/A/Artfight/dist/metadata.json (hunk 1 above)
Comment (RisingSunLight42) [reply websites/A/Artfight/dist/metadata.json:9]: Yeah, I'll change that asap

**Review by Bas950 — APPROVED** [review]

**Pull request merged**